app/docker/interceptors/containersInterceptor.js

```diff
@@ -0,0 +1,17 @@
+angular.module('portainer.app')
```

Comment:
I'm not sure how, but the interceptors code is repeating itself. Maybe there's a way to make it DRY?